modules/user/user.service.js

```diff
@@ -226,11 +231,18 @@ class UserService {
 
   async sendConfirmationLetter(email, language) {
```

Comment:
Please, add validation via `joi-validator` to check that valid email and valid language are sent

Reply:
Added